course/src/isometry.cc

```diff
@@ -0,0 +1,102 @@
+#include <cmath>
+#include <cstdint>
+#include "isometry.h"
+
+namespace ekumen {
+namespace math {
+
+Vector3::Vector3(const double& x, const double& y, const double& z) : x_(x), y_(y), z_(z) {}
+
+Vector3::~Vector3() {}
```

Comment:
If you don't need a destructor, don't add one. Unless you need to make it virtual, which is not your case here.

You'll need a constructor after addressing one of my comments, so now you can keep this one.